File: MolenApplicatie.Server/Models/Molenmaker.cs
using SQLite;

namespace MolenApplicatie.Server.Models
{
    public class MolenMaker
    {
        [PrimaryKey, AutoIncrement]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Year { get; set; }
        public int MolenDataId { get; set; }
    }
}

File: molenapplicatie.client/src/Class/MolenMaker.ts
export class MolenMaker {
  id: number;
  name: string;
  year: string;
  molenDataId: number;

  constructor(id: number, name: string, year: string, molenDataId: number) {
    this.id = id;
    this.name = name;
    this.year = year;
    this.molenDataId = molenDataId;
  }
}
